RELEASE_PLAN.md

```diff
@@ -0,0 +1,49 @@
+## Release Plan
+
+Starting with version `v2.4`, Nuxt will adhere to a formalized release plan (as good as possible).
+Also, an end of life for older major versions is defined with this document
+
+### Major versions (3.x -> 4.0)
+
+Nuxt major releases are planned every **6 months**. This depends on a few factors though:
+
+* If there are no breaking changes waiting for a release, no new major version will be published. Instead, another minor one will be released. 
+* In case of unexpected major updates of important dependencies like Vue, Webpack, and so on, major versions might be released *earlier* than planned.
+
+The goal is to provide a **migration guide** for each major version as well, as escape hatches, so existing code
+won't "just break".
+
+### Minor versions (2.1 -> 2.2)
+
+The release cycle for Nuxt.js minor versions is roughly **4 weeks**. 
+
+Three of the for weeks will be used for actual **feature implementations** while the last week will be used for 
```

Comment:
Tiny typo: `for weeks` should be `four weeks` 😊

Reply:
Thanks!